### PR TITLE
docs: add markushatvan.com

### LINKED ISSUE #1: Add markushatvan.com
I was able to finish the migration of my not-so-simple-anymore website to SvelteKit just today! 

It showcases a setup with tailwindcss JIT, mdsvex, blog posts with code snippets, contact form with svelte forms lib, RSS feed and sitemap generation, etc. 

Feel free to link to:
https://markushatvan.com
https://github.com/mhatvan/markushatvan.com

### PR DESCRIPTION
**Pull request by mhatvan**
Closes #1

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,22 +6,23 @@
...
-     - uses: [Pancakes][]
+   - uses: [Pancakes][]
 3. [SK Incognito](https://sk-incognito.vercel.app) (the unofficial SvelteKit docs) [[code](https://github.com/GrygrFlzr/kit-docs)]
-     - uses: [MDsveX][], [Tailwind CSS][]
+   - uses: [MDsveX][], [Tailwind CSS][]
 4. [Geometric Bayes](https://svelte-geometric-bayes.netlify.app) [[code](https://github.com/janosh/svelte-geometric-bayes)]
-     - uses: [MDsveX][]
+   - uses: [MDsveX][]
 5. [TikZ](https://svelte-tikz.netlify.app) [[code](https://github.com/janosh/tikz)]
+6. [markushatvan.com](https://markushatvan.com) [[code](https://github.com/mhatvan/markushatvan.com)]
```
Comment (janosh) [inline readme.md:25]: How about we add this?

```md
   - uses: [Tailwind JIT][], [MDsveX][], blog posts with code snippets, contact form with [Svelte Forms Lib][], RSS feed and sitemap
```

and then

```md
[Tailwind JIT]: https://github.com/tailwindlabs/tailwindcss-jit
[Svelte Forms Lib]: https://github.com/tjinauyeung/svelte-forms-lib
```

at the bottom?

Comment (mhatvan) [reply readme.md:25]: Done, updated the PR with another commit

Comment (janosh) [reply readme.md:25]: Excellent!

**Comment by janosh** [comment]
Thanks for the PR. Great work on your personal site, btw!

**Pull request merged**